leveling/rank.cc.lua

```diff
@@ -84,6 +90,11 @@
 			$embed.description
 		) }} {{/* If user in top 100, update the description */}}
 	{{ end }}
-
+{{ if $rankcard}}
+{{ if not $rank }} {{ $rank = urlescape "100" }} {{ end }}
```

Comment:
```suggestion
{{ $rank = or $rank "100" }}
```
`100` is already URL encoded.